docs/genai/tutorials/phi3-android.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Phi-3 for Android
+description: Tutorial for Phi-3 app for Android using the ONNX Runtime generate() API
+has_children: false
+parent: Tutorials
+grand_parent: Generate API (Preview)
+nav_order: 1
+---
+
+# Incorporate Phi-3 Model for Android Development
+This is a basic [Phi-3](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct) Android example application using [ONNX Runtime mobile](https://onnxruntime.ai/docs/tutorials/mobile/) and [ONNX Runtime Generate() API](https://github.com/microsoft/onnxruntime-genai) with support for efficiently running generative AI models. This tutorial will walk you through how to download and run the Phi-3 App on your own mobile device so you can get started incorporating Phi-3 into your own mobile developments.  
```

Comment:
Suggestion: remove the word basic

Reply:
Suggestion: download and run the Phi-3 app -> build and run the Phi-3 app

Reply:
applied suggestions